### PR TITLE
Wdfn 26

### PR DESCRIPTION
**Pull request by mbucknell**
I modeled the USGS header on the one on the USGS visual ID page. For small screen sizes, I made the logo smaller and eliminated the USGS Home link since clicking on the logo already  provides that link

### CODE REVIEW

File: assets/src/styles/header.scss [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  height:1px;
+  overflow:hidden;
+}
+
+#skip a:focus
+{
+  position:static;
+  width:auto;
+  height:auto;
+}
```
Comment (danielnaab) [inline assets/src/styles/header.scss]: Using SASS nesting for `#skip a` would be nice.

File: assets/src/styles/header.scss [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    }
+  }
+  .usgs-header-links-container {
+    font-size: $small-font-size;
+    @media screen and (max-width: $small-screen) {
+      #usgs-home-link {
+        font-size: 0px
+      }
+    }
+    @media screen and (min-width: $small-screen) {
```
Comment (danielnaab) [inline assets/src/styles/header.scss]: Minor, but looks like we have a mixin we can use here: https://github.com/18F/web-design-standards/blob/develop/src/stylesheets/core/_utilities.scss#L46-L50

File: assets/src/styles/header.scss [hunk 3]
```diff
@@ -0,0 +1,48 @@
+#skip a {
```
Comment (danielnaab) [inline assets/src/styles/header.scss]: As an inclusion file / partial, I think this file should be named `_header.scss`

File: waterdata/templates/partials/header.html [hunk 4]
```diff
@@ -1,57 +1,13 @@
...
-                </li>
-                <li>
-                    <a class="usa-nav-link" href="javascript:void(0)">
-                    <span>Distinct from each other</span>
-                    </a>
-                </li>
+        <nav class="usgs-header-links-container" role="navigation">
+            <ul>
+                <li aria-hidden="true" id="usgs-home-link"><a href="https://www.usgs.gov">USGS Home</a></li>
+                <li><a href="https://answer.usgs.gov/cgi-bin/gsanswers?tmplt=2">Contact USGS</a></li>
```
Comment (danielnaab) [inline waterdata/templates/partials/header.html]: Should this be https://answers.usgs.gov?

Comment (mbucknell) [reply waterdata/templates/partials/header.html]: Thanks. I thought I cut and pasted this correctly but it failed me again!

**Pull request merged**